An in-range update of eslint-config-standard is breaking the build 🚨

## Version **11.0.0** of [eslint-config-standard](https://github.com/standard/eslint-config-standard) was just published.

<table>
  <tr>
    <th align=left>
      Branch
    </th>
    <td>
      <a href="/cloudant/couchbackup/compare/greenkeeper%2Feslint-config-standard-11.0.0">Build failing 🚨</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Dependency
    </td>
    <td>
      eslint-config-standard
    </td>
  </tr>
  <tr>
    <th align=left>
      Current Version
    </td>
    <td>
      11.0.0-beta.0
    </td>
  </tr>
  <tr>
    <th align=left>
      Type
    </td>
    <td>
      devDependency
    </td>
  </tr>
</table>

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.


eslint-config-standard is a devDependency of this project. It **might not break your production code or affect downstream projects**, but probably breaks your build or test tools, which may **prevent deploying or publishing**.



<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push** The Travis CI build could not complete due to an error [Details](https://travis-ci.org/cloudant/couchbackup/builds/343133781?utm_source=github_status&utm_medium=notification)
- ❌ **continuous-integration/jenkins/branch** This commit is being built [Details](https://cloudant-sdks-jenkins.swg-devops.com:8443/job/couchbackup/job/greenkeeper%252Feslint-config-standard-11.0.0/1/display/redirect)
</details>


---

<details>
<summary>Commits</summary>
<p>The new version differs by 10 commits.</p>
<ul>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/c4902d20cab15971932f591d0d8cf7915ade307f"><code>c4902d2</code></a> <code>11.0.0</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/6c68c39284597f057e39708e1a9dee772a5609d2"><code>6c68c39</code></a> <code>tape@4.8</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/6d5e3f119cae5d29b4bd3127c389013c68bcb2af"><code>6d5e3f1</code></a> <code>eslint-plugin-standard@3.0.1</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/17652fef98477006313bd8dc6ce43739806c9150"><code>17652fe</code></a> <code>eslint-plugin-promise@3.6</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/0fe0f043190eb383522362d56277d3e327dbcde9"><code>0fe0f04</code></a> <code>eslint-plugin-import@2.8</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/3f0f25e7684aa52a81d9ae875229db43a084655c"><code>3f0f25e</code></a> <code>eslint@4.18</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/05a2c8e1dd3064e1b9782f09c17281ffaa449d9b"><code>05a2c8e</code></a> <code>https</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/95807618e7e3b89b437715b45360ffa40012e360"><code>9580761</code></a> <code>repo url</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/5ad91a8ffa5ae064c52e78c06cef4c2fda5a8cba"><code>5ad91a8</code></a> <code>Make "indent" rule settings explicit</code></li>
<li><a href="https://urls.greenkeeper.io/standard/eslint-config-standard/commit/e84e06dc3573aba94f876d09d164a14ab153ac05"><code>e84e06d</code></a> <code>Update URLs in package.json to use 'standard' org (#99)</code></li>
</ul>
<p>See the <a href="https://urls.greenkeeper.io/standard/eslint-config-standard/compare/3d490b951f0848d1ff86da3b5c12f14b636789a1...c4902d20cab15971932f591d0d8cf7915ade307f">full diff</a></p>
</details>


<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>


---


Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

Peer dependencies not satisfied.
```
npm ERR! peerinvalid The package eslint-config-standard@11.0.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer eslint-config-semistandard@12.0.0 wants eslint-config-standard@11.0.0-beta.0
```
Leaving `^11.0.0-beta.0` pin.

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,12 +1,12 @@
-# Unreleased
+# 2.1.0 (2018-02-20)
 
-- [IMPROVED] Enhanced resilience of backup and restore processes by enabling the
-  nodejs-cloudant retry plugin.
-- [UPGRADED] Increased nodejs-cloudant dependency to 2.x.
 - [NEW] Added API for upcoming IBM Cloud Identity and Access Management support
   for Cloudant on IBM Cloud. Note: IAM API key support is not yet enabled in the
   service.
+- [IMPROVED] Enhanced resilience of backup and restore processes by enabling the
+  nodejs-cloudant retry plugin.
 - [IMPROVED] Added URL validation for presence of host and database elements.
+- [UPGRADED] Increased nodejs-cloudant dependency to 2.x.  
 
 # 2.0.1 (2018-01-11)
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@cloudant/couchbackup",
-  "version": "2.0.2-SNAPSHOT",
+  "version": "2.1.0",
   "description": "CouchBackup - command-line backup utility for Cloudant/CouchDB",
   "homepage": "https://github.com/cloudant/couchbackup",
   "repository": "https://github.com/cloudant/couchbackup.git",

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-standard": "^3.0.0",
     "eslint-plugin-react": "^7.5.1",
-    "eslint-config-standard": "^11.0.0-beta.0",
+    "eslint-config-standard": "11.0.0-beta.0",
     "eslint-config-semistandard": "^12.0.0",
     "eslint-plugin-header": "^1.0.0",
     "jsdoc": "^3.5.2",

